presto-hive/src/main/java/com/facebook/presto/hive/OrcFileWriter.java

```diff
@@ -116,6 +117,7 @@ public OrcFileWriter(
             int[] fileInputColumnIndexes,
             Map<String, String> metadata,
             DateTimeZone hiveStorageTimeZone,
+            boolean enableExecutionBasedMemoryAccounting,
```

Comment:
This should not be introduced as a new parameter, this will break too many layers and integrations. It should be added as an option to the OrcWriterOptions.